app/models/user.py

```diff
@@ -14,11 +15,14 @@ class User(db.Model):
     created_at = db.Column(db.DateTime, default=datetime.utcnow)
     updated_at = db.Column(db.DateTime, default=datetime.utcnow)
 
+    # Add this relationship to connect with the Score model
```

Comment:
do we really need this? What is it for?
Does it mean when you query a user it will give you the scores right away?

Reply:
it is an optional code that can be used for accessing associated scores when we query users or vice versa.
I don't think we need it right now, I will drop these codes.